pkg/armhelpers/azurestack/converter.go

```diff
@@ -25,11 +25,11 @@ func DeepCopy(dst, src interface{}) error {
 	if dstValue.Type().String() != srcValue.Type().String() {
 		return fmt.Errorf("the dst type (%q) and src type (%q) are not the same", dstValue.Type().String(), srcValue.Type().String())
 	}
-	deepCopyInternal(dstValue, srcValue, 0, "")
+	deepCopyInternal(dstValue, srcValue, 0)
 	return nil
 }
 
-func deepCopyInternal(dstValue, srcValue reflect.Value, depth int, path string) {
+func deepCopyInternal(dstValue, srcValue reflect.Value, depth int) {
```

Comment:
This function was only ever called with `path=""` and `path` was otherwise unused.